acceptance/tests/reports/corrective_change_outside_puppet.rb

```diff
@@ -0,0 +1,108 @@
+require 'yaml'
+
+test_name "C98093 - a resource changed outside of Puppet will be reported as a corrective change" do
+require 'puppet/acceptance/environment_utils'
+extend Puppet::Acceptance::EnvironmentUtils
```

Comment:
The above two lines should be indented to match the other elements of the block or moved above the test block with the other require.